resources/assets/js/components/ProfileHome.vue

```diff
@@ -259,7 +269,12 @@ export default {
             // apply the profile metadata
             var person_data = metadata.data;
             this.user = person_data;
-            this.contact = person_data.primary_connection.pivot;
+            if (person_data.primary_connection) {
```

Comment:
So this is good but if we notice in `data()`, we see that `contact: null` so we don't really need the else to re-set contact to null here 😄. We should just have the if check as follows.

```suggestion
            if (person_data.primary_connection) {
                this.contact = person_data.primary_connection.pivot;
            }
```